packages/components/src/templates/next/components/internal/Notification/Notification.tsx

```diff
@@ -1,32 +1,39 @@
 "use client"
 
 import { useState } from "react"
-import { BiX } from "react-icons/bi"
+import { BiInfoCircle, BiX } from "react-icons/bi"
 
 import type { NotificationProps } from "~/interfaces"
+import { getTextAsHtml } from "~/utils"
 import BaseParagraph from "../BaseParagraph"
+import { IconButton } from "../IconButton"
 
-const NotificationBanner = ({ content }: NotificationProps) => {
+const NotificationBanner = ({ content, title }: NotificationProps) => {
   const [isShown, setIsShown] = useState(true)
   const onDismiss = () => {
     setIsShown(false)
   }
 
   return (
     isShown && (
-      <div className="flex flex-col justify-between gap-6 bg-site-primary-100 px-5 py-4 sm:flex-row sm:px-10">
-        <BaseParagraph
-          className="text-base text-content"
-          content={content}
-        ></BaseParagraph>
-        <button
-          className="ml-auto flex flex-row items-center gap-1"
-          aria-label="Dismiss notification"
-          onClick={onDismiss}
-        >
-          <p className="text-lg text-content">Dismiss</p>
-          <BiX className="h-full" />
-        </button>
+      <div className="bg-base-canvas-backdrop">
+        <div className="relative mx-auto flex max-w-screen-xl flex-row gap-4 px-6 py-8 text-base-content md:px-10 md:py-6">
```

Comment:
issue: ah missed this, the `py-8 md:py-6` means the padding is larger on mobile, is that intended?

Reply:
ya. design is like that haha